iocage/cli/create.py

```diff
@@ -98,13 +100,14 @@ def cli(release, template, count, props, pkglist, basejail, empty, short,
                 "message": "Template creation only supports TAGs!"
             })
         except ValueError:
-            ioc_common.logit({
-                "level"  : "WARNING",
-                "message": "This may be a short UUID, template creation only"
-                           " supports TAGs"
-            })
-            if not click.confirm("\nProceed?"):
-                exit()
+            if not force:
```

Comment:
yeeeaah… i did… i was looking for where this is passed, and decided, it's done by magic.
thanks for the clarification ;)